src/fiat/bank-authorities.md

```diff
@@ -83,47 +98,166 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities" \
   "modifiedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
 }
 ```
+
 **Error Responses**
 
-| Status |             Code              |                         Description                         |
-| :----- | :---------------------------- | :---------------------------------------------------------- |
-| 403    | {% break _ BANK_AUTHORITY_LIMIT_EXCEEDED %} | The account already has the max amount of bank authorities. |
+| Status |                           Code                           |                                                Description                                                 |
+| :----- | :------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------- |
+| 403    | {% break _ BANK_AUTHORITY_LIMIT_EXCEEDED %}              | The account already has the max amount of bank authorities.                                                |
 | 403    | {% break _ BANK_AUTHORITIES_FOR_BANK_ACCOUNT_EXCEEDED %} | There are already two bank authorities for the provided bank account number, which is the maximum allowed. |
 
-## Get information about a bank authority
+<span id="direct-debit-authority"></span>
+## Adding a direct debit authority to a bank account **EXPERIMENTAL**
 
-{% endpoint GET https://service.centrapay.com/api/bank-authorities/${id} %}
+{% endpoint POST https://service.centrapay.com/bank-accounts/${bankAccountId}/direct-debit-authorities %}
+
+```sh
+curl -X PUT https://service.centrapay.com/api/bank-accounts/WRhAxxWpTKb5U7pXyxQjjY/direct-debit-authorities \
+  -H "x-api-key: 1234" \
+  -H "content-type: application/json" \
+  -d '{
+    "authorized": true,
```

Comment:
Should this also contain `accountId` as we used `accountId` to asset permissions?